src/entities/Customer.java

```diff
@@ -249,24 +259,25 @@ private void checkSeats(Cinema cinema, String showtime) {
     }
 
 
-    private void addReview(ArrayList<Movie> movieDB) throws Exception {
-	System.out.println("Enter the title of the Movie you would like to add a review for");
-	BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
-	String title = reader.readLine();
-	int index = 0;
-	for (int i = 0; i < movieDB.size(); i++) {
-		if (movieDB.get(i).getTitle() == title) {
-			index = i;
-			break;
+    private void addReview(ArrayList<Movie> movieDB, MovieController mvc) throws Exception {
+		System.out.println("Enter the title of the Movie you would like to add a review for");
+		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+		String title = reader.readLine();
+		int index = 0;
+		for (int i = 0; i < movieDB.size(); i++) {
+			if (movieDB.get(i).getTitle() == title) {
+				index = movieDB.get(i).getId(); //id is the unique identifier for each movie
+				break;
+			}
 		}
-	}
-	System.out.println("Enter your rating for the movie, on a scale of 1.0 - 5.0 ");
-	int rating = Integer.parseInt(reader.readLine());
-	System.out.println("Enter your review for the movie");
-	String comment = reader.readLine();
-	Review newReview = new Review(this.username, rating, comment);
-	movieDB.get(i).addReview(newReview);
-	System.out.println("You're review was added successfully");
+		System.out.println("Enter your rating for the movie, on a scale of 1.0 - 5.0 ");
+		int rating = Integer.parseInt(reader.readLine());
+		System.out.println("Enter your review for the movie");
```

Comment:
@darrennjk might need to talk about how to do this since the implemented addRating in movie class differs from yours. So need to get your idea on this